python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -0,0 +1,274 @@
+# This file is part of meas_algorithms.
```

Comment:
The `__all__` should be after the license according to the style guide: https://developer.lsst.io/python/style.html?highlight=__all__#standard-code-order-should-be-followed

Reply:
I could have sworn I had read that differently not that long ago, but git history tells me otherwise, I feel as if I am loosing my memory or something. Thanks @timj for pointing that out.